webapp/components/interop-data.js

```diff
@@ -320,7 +320,8 @@ export const interopData = {
           { 'date': '2023-05-13', 'score': 700 },
           { 'date': '2023-06-27', 'score': 780 },
           { 'date': '2023-09-05', 'score': 860 },
-          { 'date': '2023-09-27', 'score': 870 }
+          { 'date': '2023-09-27', 'score': 870 },
+          { 'date': '2023-10-12', 'score': 910 }
```

Comment:
Another 1% merged in this morning.

```suggestion
          { 'date': '2023-09-27', 'score': 870 },
          { 'date': '2023-10-12', 'score': 910 },
          { 'date': '2023-10-13', 'score': 920 }
```